src/main/java/hivemall/ftvec/amplify/AmplifierUDTF.java

```diff
@@ -45,11 +46,11 @@ public StructObjectInspector initialize(ObjectInspector[] argOIs) throws UDFArgu
         if(!INT_TYPE_NAME.equals(argOIs[0].getTypeName())) {
             throw new UDFArgumentException("first argument must be int: " + argOIs[0].getTypeName());
         }
-        if(!(argOIs[0] instanceof WritableConstantIntObjectInspector)) {
+        if(!(argOIs[0] instanceof ConstantObjectInspector)) {
             throw new UDFArgumentException("WritableConstantIntObjectInspector is expected for the first argument: "
                     + argOIs[0].getClass().getSimpleName());
         }
-        this.xtimes = ((WritableConstantIntObjectInspector) argOIs[0]).getWritableConstantValue().get();
+        this.xtimes = ((IntWritable)((ConstantObjectInspector) argOIs[0]).getWritableConstantValue()).get();
```

Comment:
Isn't subclass of [ConstantObjectInspector](http://hive.apache.org/javadocs/r0.13.1/api/serde/org/apache/hadoop/hive/serde2/objectinspector/ConstantObjectInspector.html) that returns IntWritable was only WritableConstantIntObjectInspector?

Then, does this change make sense?
old: ((WritableConstantIntObjectInspector) argOIs[0]).getWritableConstantValue().get();
new: ((IntWritable)((ConstantObjectInspector) argOIs[0]).getWritableConstantValue()).get();

What was the exception when assuming WritableConstantIntObjectInspector?